src/frontend/komponenter/Fagsak/Vedtak/PdfFrame.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Document, Page } from 'react-pdf/dist/entry.webpack';
+import NavFrontendSpinner from 'nav-frontend-spinner';
+
+interface IPdfFrameProps {
+    pdfData: string;
```

Comment:
```suggestion
    file: string;
```

Maybe use the same name as the Document component.